builder/simplify_exprs.cc

```diff
@@ -140,6 +140,8 @@ expr simplify(const class min* op, expr a, expr b) {
       r.rewrite(min(x, ((x + c0) / c1) * c1), ((x + c0) / c1) * c1, eval(c1 > 0) && eval(c0 <= 0)) ||
 
       r.rewrite(min(x, (x / c0) * c0), (x / c0) * c0, eval(c0 > 0)) ||
+      r.rewrite(min(x, c0), c0, is_logical(x) && eval(c0 <= 0)) ||
```

Comment:
I'd put a newline between these new rules and the ones above, they're a different "category"

Also, these should have `// TODO: bounds analysis should replace these`